### DISCUSSION TITLE
"Google Chat" Question

### DISCUSSION THREAD

**Discussion by Lamf4o** [mxrch/GHunt#401] [Q&A] [answered]
I was running an attack on myself and then I noticed the "Google Chat" module, I made a research on it but didn't find anything close to an answer of my question. What exactly can you do with a Customer ID?

- Retr0

**Comment by mxrch** [comment, marked as answer]
Hi ! I don't know. 😂 I decided to show it so people can find the answer on what's the Customer ID.

What is "Retr0" ?

**Reply by Lamf4o** [reply]
Oh okay, thanks for the reply. Also what I put at the end of every message is who wrote it since We are a group, and multiple people are using this account.

- Retr0
